package.json

```diff
@@ -4,43 +4,49 @@
   "description": "A library that generates forms based on JSON schema.",
   "main": "./src/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server --inline --content-base example/ --port 8080"
   },
   "author": "Andrew Russell",
   "license": "MIT",
-  "devDependencies": {
-    "babel-core": "^6.18.0",
-    "babel-loader": "^6.2.7",
-    "babel-plugin-react-require": "^2.1.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-2": "^6.5.0",
-    "babel-root-import": "^4.0.1",
-    "css-loader": "^0.23.1",
-    "file-loader": "^0.9.0",
-    "node-sass": "^3.7.0",
-    "sass-loader": "^3.2.0",
-    "style-loader": "^0.13.1",
-    "url-loader": "^0.5.8",
-    "webpack": "^1.13.1"
-  },
   "dependencies": {
-    "babel-core": "^6.10.4",
-    "babel-generator": "^6.11.0",
-    "babel-polyfill": "^6.16.0",
     "classnames": "^2.2.5",
     "google-libphonenumber": "^1.0.27",
     "immutable": "^3.8.1",
     "moment": "^2.14.1",
-    "react": "^15.3.2",
     "react-currency-input": "^1.0.4",
     "react-date-picker": "^5.3.27",
-    "react-dom": "^15.3.2",
     "react-dropzone": "^3.9.2",
     "react-immutable-proptypes": "^2.1.0",
     "react-maskedinput": "^3.2.4",
     "react-rte": "^0.11.0",
     "react-select": "^1.0.0-rc.1",
     "react-tabs": "^0.7.0"
+  },
+  "peerDependencies": {
+    "react": "^15.0.0"
```

Comment:
For sures, peerDependenices are for dependencies that are expected to be used by the code importing this module. You those to have a loose version restriction version since that restriction will apply to the person using your code.

We want it in dev dependencies because we use it when running the example code, and in that case we just take the latest version of react (just like a person using our code would).